src/pages/PrintOnlyScreen.tsx

```diff
@@ -92,7 +54,7 @@ const PrintOnlyScreen = ({
   const printBallotTimer = useRef(0)
   const printBallotTrackingCodeTimer = useRef(0)
 
-  const [stateMachine, sendStateMachineEvent] = useMachine(printStateMachine)
+  const [printingState, setPrintingState] = useState('')
```

Comment:
Use an enum for these values? Makes it less likely we accidentally typo them:

```
enum PrintingState {
  PrintBallot = 'PRINT_BALLOT'
  …
}
```

Names and values are up to you.

Reply:
I need to do some fine tuning of this anyways for error conditions, so I will do that in a follow-on PR, but want to get a build going right now so merging prior to this fix.